packages/grafana-ui/tsconfig.json

```diff
@@ -3,7 +3,8 @@
     "baseUrl": "./",
     "declarationDir": "dist",
     "outDir": "compiled",
-    "rootDirs": [".", "stories"]
+    "rootDirs": [".", "stories"],
+    "jsx": "react"
```

Comment:
Wonder why is this necessary?

Reply:
Line 7 was added to silence this error message:
![Screenshot 2022-05-30 at 15 43 47](https://user-images.githubusercontent.com/48948963/171004975-c2748b21-4203-4d8f-a127-a0f074c3b7ae.png)
As this is only a warning and it's not breaking if I remove it, I can also remove it.